pkg/controller/tailoredprofile/tailoredprofile_controller.go

```diff
@@ -333,6 +337,102 @@ func (r *ReconcileTailoredProfile) ensureConfigMapOutputObject(tp *compliancev1a
 	return reconcile.Result{}, nil
 }
 
+func (r *ReconcileTailoredProfile) ensurePolicyOutputObject(tp *compliancev1alpha1.TailoredProfile, tpcm *corev1.ConfigMap, pb *compliancev1alpha1.ProfileBundle, logger logr.Logger) (reconcile.Result, error) {
+	objKey := types.NamespacedName{Name: tp.GetName(), Namespace: tp.GetNamespace()}
+	// reset namespace
+	tpcm.SetNamespace("")
+	cmUnstructured, err := runtime.DefaultUnstructuredConverter.ToUnstructured(tpcm)
+	if err != nil {
+		updateErr := r.updateTailoredProfileStatusError(
+			tp,
+			fmt.Errorf("Couldn't convert ConfigMap to Unstructured: %s", err),
+		)
+		return reconcile.Result{}, updateErr
+	}
+
+	// TODO(jaosorior): Import library and use actual object instead of unstructured
+	suiteObj := map[string]interface{}{
+		"apiVersion": "compliance.openshift.io/v1alpha1",
+		"kind":       "ComplianceSuite",
+		"metadata": map[string]interface{}{
+			"name": objKey.Name,
+		},
+		"spec": map[string]interface{}{
+			"schedule":              "0 1 * * *",
+			"autoApplyRemediations": false,
+			"scans": []interface{}{
+				map[string]interface{}{
+					"name":         objKey.Name + "-worker-scan",
+					"profile":      xccdf.GetXCCDFProfileID(tp),
+					"content":      pb.Spec.ContentFile,
+					"contentImage": pb.Spec.ContentImage,
+					"nodeSelector": map[string]interface{}{
+						"node-role.kubernetes.io/worker": "",
```

Comment:
This is just for the PoC, right? Or otherwise why only scan workers?

Reply:
Yep, it's only for the PoC